### ISSUE TITLE
RenderContainer: Warning: unstable_flushDiscreteUpdates: Cannot flush updates when React is already rendering.

### ISSUE THREAD

**Issue by ildar-meyker** [skbkontur/retail-ui#2872]
`<Kebab size="large">
						<MenuItem
							icon={<EyeOpened />}
							onClick={() => console.log(1)}
						>
							Все
						</MenuItem>
					</Kebab>`

При клике на MenuItem бросает ошибку Warning: unstable_flushDiscreteUpdates: Cannot flush updates when React is already rendering.

**Comment by JackUait** [comment]
Привет! Спасибо за ишью. Похоже, что проблема очень старая. Получилось отследить её до `3.0.0`, но очевидно, что она появилась раньше. Сейчас у нас есть две задачи, одна из которых потенциально может закрыть проблему, это задача про вложенность `MenuItem` из #2749, а также вторая задача, которая наверняка закроет эту проблему, это задача про перевод (и рефактор) всей библиотеки на функциональные компоненты. Первая задача будет закрыта в ближайшее время, про вторую тоже самое сказать не могу.

Подскажи, насколько критичен для вас этот баг? Если баг критичен для тебя или твоей команды, возьмём этот баг как отдельную задачу.

**Comment by dzekh** [comment]
@ildar-meyker а почему важная штука? Падает ли приложение от этой ошибки?

**Comment by ildar-meyker** [comment]
@dzekh нет, не падает. Природа ошибки мне не ясна, не исследовал, и возможные последствия тоже. Компонент активно используется, поэтому важность обозначена лишь на уровне предположения.

**Comment by dzekh** [comment]
@JackUait @zhzz А эта задачка не связана с #2941 ? может она там исправлена?
